### PR TITLE
Define default transition and add editorconfig

### PR DESCRIPTION
**Pull request by gabrieluizramos**
There is a slightly difference between the sidebars and the main content (post) transitions.

As you can see at:
Screen Recording 2019-07-23 at 13 37 35

This PR creates some default transitions and applies it at the sidebars and main content containers.

Giving them a default transition like:
Screen Recording 2019-07-23 at 13 37 10

### CODE REVIEW

File: src/styles/transitions.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const defaultTiming = '0.5s';
+const bgTransition = `background ${defaultTiming}`;
+const colorTransition = `color ${defaultTiming}`;
+const defaultTransition = `${bgTransition}, ${colorTransition}`;
+
+export default {
+    DEFAULT: defaultTransition,
+    COLOR: colorTransition,
+    BACKGROUND: bgTransition
+};
```
Comment (willianjusten) [inline src/styles/transitions.js]: Add a newline please =)
Isn't prettier working with you?

Comment (gabrieluizramos) [reply src/styles/transitions.js]: Sure!
It works. Actually, I forgot to run the format 😅 

I'm also adding a .editorconfig file on this PR :)

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Editorconfig: https://editorconfig.org/
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
```
Comment (willianjusten) [inline .editorconfig]: Nooooooooo, spaces >>>> tab

Comment (willianjusten) [reply .editorconfig]: <3

Comment (gabrieluizramos) [reply .editorconfig]: 🤣 done

**Comment by willianjusten** [comment]
Just tested, everything looks amazing! <3
Thaaaaaanks!

**Comment by gabrieluizramos** [comment]
@willianjusten 🤘

**Pull request merged**